style.css

```diff
@@ -284,3 +284,32 @@ body {
     max-width: 120rem;
     width: 100%;
 }
+
+
+/* -------------------  AUTHENTICATION  ----------------- */
+
+/* LATER PUT THIS IN UTILITY CLASS */
+.font-black{
+    color: var(--OFF-BLACK);
+}
+
+.auth-container{
+    display: flex;
```

Comment:
all these properties too can be clubbed under one class like you did with margin and padding.

Reply:
Yup understood what you trying to say but here I need to use more than 4-5 utilities so made a seperate component so that it doesn't clutter file with many class names.